ssl/tls_srp.c

```diff
@@ -105,7 +107,7 @@ int SSL_SRP_CTX_init(struct ssl_st *s)
     s->srp_ctx.b = NULL;
     s->srp_ctx.v = NULL;
     s->srp_ctx.login = NULL;
-    s->srp_ctx.info = ctx->srp_ctx.info;
+    s->srp_ctx.info = NULL;
```

Comment:
Probably a memset() of srp_ctx is more appropriate here, rather than all these NULL assignments